types/redux-shortcuts/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for redux-shortcuts 0.0.3
+// Project: https://github.com/nak2k/node-redux-shortcuts
+// Definitions by: Tim Stirrat <https://github.com/tstirrat>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+import { ActionCreator, Action, Dispatch } from "redux";
+// import * as Mousetrap from 'mousetrap';
```

Comment:
Why is this commented out?

Reply:
I couldn't figure out how to re-export the `import * as Mousetrap`, nor the `MousetrapInstance`. Though I'll give it another shot

Reply:
Thanks, that worked.